loleaflet/src/control/Control.LokDialog.js

```diff
@@ -765,10 +765,14 @@ L.Control.LokDialog = L.Control.extend({
 		}
 
 		var newHeight = height * scale;
+		// console.log('newHeight = height * scale: ' + newHeight + ' = ' + height + ' * ' + scale);
+		// console.log('if (canvas.height != newHeight) : if(' + canvas.height + ' != ' + newHeight + ')');
+		// console.debug(canvas);
```

Comment:
You probably didn't mean to commit commented out debug code. ;)

Otherwise, the patch works well for me, thanks!

Reply:
Well It was I who added those in the first place, I just thought it would be better to commented them out and if someone wants to check those values can always uncommented but on the 2nd thought I will be removing those

Reply:
Yes, that's what I meant, it was added with the commit, and was added as commented out. I think providing hints for debugging is a great idea, the way I would probably consider doing it is in a more descriptive and explanatory way rather than raw debug code. But those are just my two cents, and removing the code is good, too, thanks for updating the pull request!